examples/llm/benchmarks/README.md

```diff
@@ -68,18 +67,57 @@ Note: Check the `disagg.log` to make sure the service is fully started before co
 
 Collect the performance numbers as shown on the [Collecting Performance Numbers](#collecting-performance-numbers) section below.
 
+## Disaggregated Multi Node Benchmarking
+
+*Two H100 80GB x8 nodes are required for this setup.*
```

Comment:
```suggestion
*Two H100 80GB x8 nodes are required for this setup.*

> [!Note]: 
> Nodes used for benchmarking were part of a cluster connected via > InfiniBand NDR with 8 connections for compute and 2 for storage. > Both fabrics were on their own fat tree non-blocking topology. 
```

Please verify the look of this in the readme - but lets a add anote on the networking configuration used for multi-node